core/v2/serialization.go

```diff
@@ -129,6 +136,7 @@ func (b *BlobHeader) BlobKey() (BlobKey, error) {
 			},
 			DataLength: uint32(b.BlobCommitments.Length),
 		},
+		b.Salt,
```

Comment:
is it normal that blob_header_hash doesn't contain the payment_header nor the signature?

This also means that the salt was not even part of this hash until now? Seems like that was a bug/mistake?

Reply:
I think we agreed that `blob_header_hash` doesn't contain payment header nor the signature

Salt was part of the payment metadata hash, so it didn't need to be here

Reply:
gonna go ahead and merge since I think we have agreed on the structures. please let me know if you have more concerns and I will create subsequent PRs addressing them